spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-config-sample/README.adoc

```diff
@@ -0,0 +1,78 @@
+= Spring Cloud GCP Pub/Sub Configuration Management Code Sample
```

Comment:
How about just "Spring Cloud Bus Code Sample" for the title?

Reply:
Given the Vision sample/demo conversation, does it make sense to name it "Spring Cloud Bus Configuration Management Demo"? Or still too verbose?

Reply:
Sure, that works for me.